src/settings.ts

```diff
@@ -12,14 +12,16 @@ export class RDSettingTab extends PluginSettingTab {
 
 	display(): void {
 		const { containerEl } = this;
+		const client_name = "Obsidian Readeck Importer";
+
 
 		containerEl.empty();
 
 		const loggedIn = this.plugin.settings.apiToken !== "";
 
 		new Setting(containerEl)
 			.setName('API URL')
-			.setDesc('URL of Readeck instance (without trailing "/")')
+			.setDesc('URL of Readeck instance (without trailing "/"). E.g. https://readeck.domain.tld')
```

Comment:
It's your job to cleanup the provided URL and take into account that a Readeck instance can be in a sub folder.
See https://codeberg.org/readeck/browser-extension/src/commit/de98607d9f76d2422dd27c3d3be0f99b704a2863/src/app/login.svelte#L69-L72